karma.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: karma
+  version: "0.120"
+  description: "A dashboard for managing alerts from Alertmanager"
+  epoch: 0
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prymitive/karma.git
+      tag: v${{package.version}}
+      expected-commit: 8009f4aa581980654e5c67b26b41e85d9e7ccce0
+
+  - name: install-npm-dependencies
+    working-directory: ui
+    pipeline:
+      - runs: |
+          mkdir -p node_modules/.bin
+          touch node_modules/.install
+          npm install --package-lock-only
+          npm ci
+          cat node_modules/bootstrap/scss/_root.scss | sed s/':root {'/'* {'/ > src/Styles/BootstrapRoot.scss
```

Comment:
Can we also add a new line above, (just after `npm ci`)
```
npm prune --omit=dev
```

To prune unnecessary things?